test/math.jl

```diff
@@ -1008,7 +1008,11 @@ using .Main.Furlongs
         @test @inferred hypot(i, i, i, i) ≈ 2.0i
         @test @inferred hypot(i//1, 1//i, 1//i) ≈ i
     end
-    @test @inferred hypot(typemin(Int), typemin(Int)) ≈ -sqrt(2)typemin(Int)
+    let i = typemin(Int)
+        @test @inferred hypot(i, i) ≈ -√2i
```

Comment:
I think you want `@test @inferred(hypot(i, i)) ≈ -√2i` here since presumably you want to test specifically that `hypot` is inferred, not the entire expression `hypot(i, i) ≈ -√2i`.